spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -303,10 +303,8 @@ public SparkContext createSparkContext() {
     for (Object k : intpProperty.keySet()) {
       String key = (String) k;
       String val = toString(intpProperty.get(key));
-      if (!key.startsWith("spark.") || !val.trim().isEmpty()) {
-        logger.debug(String.format("SparkConf: key = [%s], value = [%s]", key, val));
-        conf.set(key, val);
-      }
```

Comment:
:+1: that code by @jongyoul actually fixed a nasty bug we had with empty interpreter values from GUI